src/Veldrid/MTL/MTLGraphicsDevice.cs

```diff
@@ -8,6 +8,10 @@
 using NativeLibraryLoader;
 using Veldrid.MetalBindings;
 
+#if NET5_0_OR_GREATER
+using NativeLibrary = NativeLibraryLoader.NativeLibrary;
```

Comment:
I don't think this needs to be behind an `#if`. It should be harmless to include everywhere.